examples/conditional-exit/example.py

```diff
@@ -21,7 +23,12 @@ def someone_says_the_magic_word(events):
         return True
 
     # This will run only until someone enters the magic word(s) into a channel.
-    app.run(until=someone_says_the_magic_word)
+    try:
+      app.run(until=someone_says_the_magic_word)
+    except MissingToken:
+      sys.exit('Unable to find Slack API token. \nPlease make sure you have '\
+               'obtained one from https://api.slack.com/docs/token-types#bot.')
```

Comment:
The link here describes what a bot token is, but doesn't actually describe the process of obtaining one. It would probably be better to link to some Slack documentation on how to acquire a token or just to say
> Please read more on the available token types at https://api.slack.com/docs/token-types.